pyei/plot_utils.py

```diff
@@ -15,7 +15,6 @@ def plot_precincts(voting_prefs_group1, voting_prefs_group2, y_labels=None, ax=N
         fig, ax = plt.subplots()
     x = np.linspace(0, 1, n_x_pts)
 
-    N = voting_prefs_group1.shape[1]
```

Comment:
Deleted this line because it seemed redundant with line 14

Reply:
Yes - thanks! This reminds me that I should give this variable a better name too!